src/TropicalGeometry/groebner_polyhedron.jl

```diff
@@ -0,0 +1,48 @@
+###
+# Computing (tropical) Groebner polyhedra in Oscar
+# ================================================
+#
+# For a definition of tropical Groebner cones see Section 2.5 in:
+#   D. Maclagan, B. Sturmfels: Introduction to tropical geometry
+# To see how they can be computed using standard bases see:
+#   T. Markwig, Y. Ren: Computing tropical varieties over fields with valuation
+###
+
+
+#=======
+tropical Groebner polyhedron
+todo: proper documentation
+Example:
+
+val_2 = ValuationMap(QQ,2)
+Kx,(x,y,z) = PolynomialRing(QQ,3)
+w = [0,0,0]
+I = ideal([x+2*y,y+2*z])
+groebner_polyhedron(I,val_2,w)
+
+Kt,t = RationalFunctionField(QQ,"t")
+val_t = ValuationMap(Kt,t)
+Ktx,(x,y,z) = PolynomialRing(Kt,3)
+w = [0,0,0]
+I = ideal([x+t*y,y+t*z])
+groebner_polyhedron(I,val_t,w)
+=======#
+function groebner_polyhedron(I,val::ValuationMap{K,p} where {K,p},w::Vector{Int})
```

Comment:
Also here and in a few more places
```suggestion
function groebner_polyhedron(I, val::ValuationMap, w::Vector{Int})
```